universum/lib/utils.py

```diff
@@ -57,36 +57,44 @@ def detect_environment() -> str:
     """
     :return: "tc" if the script is launched on TeamCity agent,
              "jenkins" is launched on Jenkins agent,
+             "github" is launched on Github Actions agent,
```

Comment:
I'm not sure "GitHub Actions agent" is a correct statement. Do they use "agents" as well?